### PR TITLE
[dev-2.4] March 19th k3s releases

### PR DESCRIPTION
**Pull request by MonzElmasry**
This pr adds the new patch versions for k3s and rke2
k3s issues :  k3s-io/k3s#2975 and k3s-io/k3s#2976

### CODE REVIEW

**Review by davidnuzik — APPROVED** [review]
LGTM thanks!
Note to self or anyone: Let's wait until the releases are out before merging :)

**Review by kinarashah — APPROVED** [review]

**Pull request merged**
